CHANGELOG.md

```diff
@@ -24,8 +24,6 @@
 
 * ![Feature][badge-feature] Deployment is now more customizable and thus not as tied to Travis CI as before. ([#1147][github-1147], [#1171][github-1171])
```

Comment:
Mark this as experimental too?

Reply:
I'd say that it's only the GitHub Actions part that needs to be experimental. The rest of the refactoring is still good even if we'd nuke the Actions part.

Reply:
Right, but I wanted to redo that interface to properly support PR previews :p

Reply:
Actually, I have that almost finished on some branch. Let me push that and we can see.

Reply:
#1180